<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>771</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(190, 225, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_educational_information">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Educational Information ðŸŽ“ </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_institution">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 218, 30);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>Your institution</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_UGCourse">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 218, 30);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>UG Course</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_resume">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 218, 30);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>Your resume</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(170, 170, 255);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255);

</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_input_UGCourse">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <item>
     <property name="text">
      <string>Select</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B .A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B .Com</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B .Tech</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BSc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B .Arch</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_ug_cgpa">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 218, 30);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>CGPA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_PGCourse">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 218, 30);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>PG Course</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_input_PGCourse">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <item>
     <property name="text">
      <string>Select</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M .A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M .Com</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M .Tech</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MSc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MS</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_pg_cgpa">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>320</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 218, 30);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>CGPA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input_institution">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
padding:5px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Type your institution name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input_resume">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
padding:5px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Upload your resume from this device</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input_pg_cgpa">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>320</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input_ug_cgpa">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_folder">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>440</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding:5px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>ðŸ“‚</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
